pkg/server/handler/v1alpha1/tenant.go

```diff
@@ -117,49 +178,64 @@ func CreateAdminTenant() error {
 		core_v1.ResourceRequestsMemory: common.QuotaMemoryRequest,
 	}
 
-	tenant := &apiv1alpha1.Tenant{
-		Metadata: apiv1alpha1.TenantMetadata{
+	tenant := &api.Tenant{
+		Metadata: api.Metadata{
 			Name: common.AdminTenant,
 		},
-		Spec: apiv1alpha1.TenantSpec{
+		Spec: api.TenantSpec{
 			// TODO(zhujian7), read from configmap
-			PersistentVolumeClaim: apiv1alpha1.PersistentVolumeClaim{
+			PersistentVolumeClaim: api.PersistentVolumeClaim{
 				StorageClass: "", // use default storageclass
 				Size:         common.DefaultPVCSize,
 			},
 			ResourceQuota: quota,
 		},
 	}
 
-	return createCycloneTenant(tenant)
+	return createTenant(tenant)
+}
+
+func createControlClusterIntegration(tenant string) error {
+	in := &api.Integration{
+		Metadata: api.Metadata{
+			Name:        common.ControlClusterName,
```

Comment:
I think there is no need to create control cluster, use can directly in default.

Reply:
I don't think so, as control cluster could be closed by users, so it is necessary to hold an instance(secret) at the backend.